src/main/java/de/mediathekview/mserver/crawler/hr/tasks/HrSendungsfolgedetailsTask.java

```diff
@@ -2,58 +2,199 @@
-import de.mediathekview.mserver.crawler.basic.AbstractDocumentTask;
-import de.mediathekview.mserver.crawler.basic.AbstractUrlTask;
-import de.mediathekview.mserver.crawler.basic.CrawlerUrlDTO;
-import mServer.crawler.CrawlerTool;
+import org.jsoup.select.Elements;
 
 public class HrSendungsfolgedetailsTask extends AbstractDocumentTask<Film, CrawlerUrlDTO> {
+
   private static final String ATTRIBUTE_DATETIME = "datetime";
+  private static final String ATTRIBUTE_VIDEO_JSON = "data-hr-video-adaptive";
   private static final Logger LOG = LogManager.getLogger(HrSendungsfolgedetailsTask.class);
   private static final long serialVersionUID = 6138774185290017974L;
-  private static final String THEMA_SELECTOR = ".c-programHeader__headline.text__headline";
-  private static final String TITEL_SELECTOR = ".c-programHeader__subline.text__topline";
-  private static final String DATE_TIME_SELECTOR = ".c-programHeader__mediaWrapper time";
-  private static final String DAUER_SELECTOR = ".c-programHeader__mediaWrapper .mediaInfo__byline";
-  private static final String BESCHREIBUNG_SELECTOR = ".copytext__text.copytext__text strong";
-  private static final String VIDEO_URL_SELECTOR = ".c-programHeader__mediaWrapper source";
-  private static final String UT_URL_SELECTOR = ".c-programHeader__mediaWrapper track";
+  private static final String THEMA_SELECTOR1 = ".breadcrumbNav__item span[itemprop=title]";
+  private static final String THEMA_SELECTOR2 = ".c-programHeader__headline.text__headline";
+  private static final String TITLE_SELECTOR1 = ".c-programHeader__subline.text__topline";
+  private static final String TITLE_SELECTOR2 = ".c-contentHeader__headline";
+  private static final String DATE_TIME_SELECTOR1 = ".c-programHeader__mediaWrapper time";
+  private static final String DATE_TIME_SELECTOR2 = ".c-contentHeader__lead time";
+  private static final String DAUER_SELECTOR1 = ".c-programHeader__mediaWrapper .mediaInfo__byline";
+  private static final String DAUER_SELECTOR2 = ".c-contentHeader .mediaInfo__byline";
+  private static final String DESCRIPTION_SELECTOR1 = ".copytext__text.copytext__text strong";
+  private static final String DESCRIPTION_SELECTOR2 = ".copytext__text";
+  private static final String VIDEO_URL_SELECTOR1 = ".c-programHeader__mediaWrapper source";
+  private static final String VIDEO_URL_SELECTOR2 = "div.videoElement";
+  private static final String UT_URL_SELECTOR1 = ".c-programHeader__mediaWrapper track";
+  private static final String UT_URL_SELECTOR2 = ".c-contentHeader__lead track";
+
+  private static final Type OPTIONAL_ARDVIDEOINFODTO_TYPE_TOKEN = new TypeToken<Optional<ArdVideoInfoDTO>>() {
+  }.getType();
+
+  private final Gson gson;
 
   public HrSendungsfolgedetailsTask(final AbstractCrawler aCrawler,
-      final ConcurrentLinkedQueue<CrawlerUrlDTO> aUrlToCrawlDTOs) {
-    super(aCrawler, aUrlToCrawlDTOs);
+      final ConcurrentLinkedQueue<CrawlerUrlDTO> aUrlToCrawlDtos) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The type of the "aUrlToCrawlDtos" object should be an interface such as "Queue" rather than the implementation "ConcurrentLinkedQueue". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)